### PR TITLE
Update printoperator.md

### PR DESCRIPTION
**Pull request by AMK2805**
Corrected statements used in print table operator page

### CODE REVIEW

**Comment by PRMerger18** [comment]
@AMK2805 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/query/printoperator.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.date: 03/16/2019
 ---
 # print operator
 
-Outputs single-row with one or more scalar expressions.
+Outputs single-row with one or more scalar expression results as columns.
```
Comment (slavikn84) [inline data-explorer/kusto/query/printoperator.md:10]:
```suggestion
Outputs a single-row with one or more scalar expression results as columns.
```

File: data-explorer/kusto/query/printoperator.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ print x=1, s=strcat("Hello", ", ", "World!")
 
 ## Returns
 
-A single-column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.
+A one or more column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.
```
Comment (slavikn84) [inline data-explorer/kusto/query/printoperator.md:28]: Needs rephrasing, it's not a "single cell" anymore :)
By the way, the beginning of the sentence is "A one or more column" sounds a bit strange (though grammatically it's probably correct).

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/printoperator.md (hunk 2 above)
Comment (YaelSchuster) [inline data-explorer/kusto/query/printoperator.md:28]:
```suggestion
A table with one or more columns and a single row, with the value of the evaluated *ScalarExpression*.
```

**Comment by YaelSchuster** [comment]
New PR on private repo has been created to update wording,

#please-close

**Pull request closed without merging**